presto-elasticsearch/src/test/java/io/prestosql/elasticsearch/TestElasticsearchIntegrationSmokeTest.java

```diff
@@ -102,105 +112,100 @@ public void testNestedFields()
-                                "    }" +
-                                "  }," +
-                                "  \"properties\":{" +
-                                "    \"a\": {" +
-                                "      \"type\": \"object\"," +
-                                "      \"properties\": {" +
-                                "        \"b\": {" +
-                                "          \"type\": \"object\"," +
-                                "          \"properties\": {" +
-                                "            \"x\": {" +
-                                "              \"type\": \"integer\"" +
-                                "            }," +
-                                "            \"y\": {" +
-                                "              \"type\": \"keyword\"" +
-                                "            }" +
-                                "          } " +
-                                "        }" +
-                                "      }" +
-                                "    }," +
-                                "    \"c\": {" +
-                                "      \"type\": \"object\"," +
-                                "      \"properties\": {" +
-                                "        \"d\": {" +
-                                "          \"type\": \"keyword\"" +
-                                "        }," +
-                                "        \"e\": {" +
-                                "          \"type\": \"keyword\"" +
-                                "        }," +
-                                "        \"f\": {" +
-                                "          \"type\": \"object\"," +
-                                "          \"properties\": {" +
-                                "            \"g\": {" +
-                                "              \"type\": \"integer\"" +
-                                "            }," +
-                                "            \"h\": {" +
-                                "              \"type\": \"integer\"" +
-                                "            }" +
-                                "          } " +
-                                "        }" +
-                                "      }" +
-                                "    }," +
-                                "    \"i\": {" +
-                                "      \"type\": \"long\"" +
-                                "    }," +
-                                "    \"j\": {" +
-                                "      \"type\": \"long\"" +
-                                "    }," +
-                                "    \"k\": {" +
-                                "      \"type\": \"long\"" +
-                                "    }" +
-                                "  }" +
-                                "}",
-                        XContentType.JSON)
-                .get();
+        String mapping = "" +
```

Comment:
Can we use `//launguage=JSON` comment? As you may already know, it will improve the readability on Intellij.
<img width="378" alt="Screen Shot 2020-01-23 at 11 55 38" src="https://user-images.githubusercontent.com/6237050/72953100-60d42400-3dd7-11ea-80e6-9f1c1d866b56.png">

Reply:
That’s a good idea. I’ll see if the `@Language` annotation works.